foursight_core/react/api/react_api.py

```diff
@@ -451,6 +452,9 @@ def _create_user_record_from_input(self, user: dict, include_deletes: bool = Fal
                 del user["role"]
             if "roles" in user:
                 del user["roles"]
+            if "status" in user:
+                if not user["status"] or user["status"] == "-":
+                    del user["status"]
```

Comment:
You are modifying an input argument. Do you mean to do that? It's far safer to make a copy of the arg when you receive it so you don't change someone else's data. Side-effecting a caller's constants is a bad idea.